packages/cli/src/utils/createNextjsPages.ts

```diff
@@ -0,0 +1,64 @@
+import path from 'path'
+import fs from 'fs'
+
+import { withBasePath } from './directory'
+import { myAccountPageTemplate } from './templates/myAccountPage'
+
+const ALLOWED_PREFIX_PAGES = ['/account']
+
+const createExternalPages = (
+  customizationPagesDir: string,
+  corePagesDir: string,
+  baseCustomizationPagesDir: string
+) => {
+  fs.readdirSync(customizationPagesDir).forEach((file) => {
+    const filePath = path.join(customizationPagesDir, file)
+    const destinationPath = path.join(corePagesDir, file)
+    if (fs.statSync(filePath).isDirectory()) {
+      if (!fs.existsSync(destinationPath)) {
+        fs.mkdirSync(destinationPath, { recursive: true })
+      }
+      return createExternalPages(
+        filePath,
+        destinationPath,
+        baseCustomizationPagesDir
+      )
+    }
+
+    if (file.endsWith('.tsx') && !fs.existsSync(destinationPath)) {
```

Comment:
here is the magic, right? If the file already exists in the destination (like the default `/account/index.tsx`), we don't create it.

Reply:
Yeah. should not be possible to create the my account fixed pages (/account/profile, /account/orders, ...)